src/FileSystemUtilities.js

```diff
@@ -51,6 +51,17 @@ export default class FileSystemUtilities {
     rimraf(filePath, callback);
   }
 
+  @logger.logifyAsync
+  static symlink(src, dest, callback, type = "file") {
```

Comment:
It doesn't look like the caller uses the `type` parameter?

All other asynchronous methods in `FileSystemUtilities` take the callback as the _final_ argument.